From 322e45d390211ffb42b9916b3a4ffe54b73df0f7 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 22 Nov 2023 13:05:00 +0000
Subject: [PATCH] gpio-poweroff: Disable the WARN

With the new support for a chain of sys_off handlers, gpio-poweroff
does not disable a normal shutdown (though it does delay it). There
is therefore no need for the noisy WARN from the kernel.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/power/reset/gpio-poweroff.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/power/reset/gpio-poweroff.c
+++ b/drivers/power/reset/gpio-poweroff.c
@@ -44,7 +44,7 @@ static int gpio_poweroff_do_poweroff(str
 	/* give it some time */
 	mdelay(gpio_poweroff->timeout_ms);
 
-	WARN_ON(1);
+	//WARN_ON(1);
 
 	return NOTIFY_DONE;
 }
